### ISSUE TITLE
Open Asset Import Library (assimp) plugin for [model]

### ISSUE THREAD

**Issue by 60-hz** [pd-l2ork/pd-l2ork#9]
Since Gem is working quite well now, it would be great to be able to support assimp loader for [model] which allows hundred of file format.

It works well under MacOs and Windows from here https://git.iem.at/pd/Gem/-/jobs/

I guess the library should be installed before Gem configure https://github.com/assimp/assimp

**Comment by pd-l2ork** [comment]
Thanks for your feedback. Does upstream Gem support this out-of-box? Also, is assimp library available as a prebuilt library for mingw, brew (OSX), and Linux?

**Comment by umlaeute** [comment]
>  Does upstream Gem support this out-of-box?

yes (Gem will detect and use `assimp` when it's installed at configure time)

> Also, is assimp library available as a prebuilt library for mingw, brew (OSX), and Linux?

- https://packages.msys2.org/base/mingw-w64-assimp
- https://formulae.brew.sh/formula/assimp
- https://repology.org/project/assimp/versions
